packages/skeleton-svelte/src/lib/components/Progress/Progress.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
```

Comment:
FYI the React component appeared higher in the the list when reviewing. Please make sure to echo all relevant changes to the Svelte side too please!